src/coreclr/src/jit/compiler.cpp

```diff
@@ -1299,7 +1299,7 @@ size_t genFlowNodeCnt;
 
 #ifdef DEBUG
 /* static */
-unsigned Compiler::s_compMethodsCount = 0; // to produce unique label names
+LONG Compiler::s_compMethodsCount = 0; // to produce unique label names
```

Comment:
have not seen many `LONG` in the jit, I am not sure in which case we use them.

Reply:
@kunalspathak Presumably it chosen because it matches the signature of InterlockedIncrement

Reply:
That's correct.